scripts/SANS/sans/user_file/user_file_parser.py

```diff
@@ -894,6 +894,9 @@ def _is_line_mask(self, line):
     def _is_time_mask(self, line):
         return does_pattern_match(self._time_pattern, line)
 
+    def _is_time_mask_2(self, line):
```

Comment:
This function doesn't appear to be used anywhere (and as far as I can tell `_time_pattern_2` doesn't exist - is the plan to use this in future?